vm-systemd/qubes-network-uplink@.service

```diff
@@ -3,6 +3,8 @@ Description=Qubes network uplink (%i) setup
 After=network-pre.target qubes-iptables.service
 After=sys-subsystem-net-devices-%i.device
 BindsTo=sys-subsystem-net-devices-%i.device
+After=systemd-sysctl.service
```

Comment:
This is already implicitly here (`systemd-sysctl.service` has `Before=sysinint.target` and everything else has `After=sysinit.target` unless `DefaultDependencies=no`).

Reply:
:+1: